internal/k8shandler/metric_exporter.go

```diff
@@ -0,0 +1,61 @@
+package k8shandler
```

Comment:
We have been making effort to do away with the catch all of k8shandler.  Please relocate to either: internal/metrics/logfilemetricexporter or internal/logfilemetricexporter

In general, earlier refactorings were trying to apply readability like: `elasticsearch.Reconcile` or `reconcile.Service`